ui/src/app/applications/components/applications-list/applications-list.tsx

```diff
@@ -40,6 +40,16 @@ const APP_FIELDS = [
 const APP_LIST_FIELDS = ['metadata.resourceVersion', ...APP_FIELDS.map(field => `items.${field}`)];
 const APP_WATCH_FIELDS = ['result.type', ...APP_FIELDS.map(field => `result.application.${field}`)];
 
+document.onkeypress = e => {
```

Comment:
Overriding onkeypress might be unpleasant if we add a key press handler in another component. 
Please use useEffect hook to add handler with `document.addEventListener` and remove listener in hook cleanup. E.g.

```
useEffect(() => {
    const handleKeyPress = (e) => {
       ...
    }
    document.addEventListener('keypress', handleKeyPress);
    return () => {
      document.removeEventListener('keypress', handleKeyPress);
    };
  });
```

Reply:
I've updated the code to use `React.useEffect`

Reply:
We use `/` while adding the git repository in the application create form. We shouldn't consider `/` as a special shortcut when the Application form is opened.

Reply:
@alexmt I've added a check to disable this shortcut when the new application form is opened.